src/pods/test-verb-forms/test-verb-forms.component.tsx

```diff
@@ -66,9 +80,9 @@ export const TestVerbFormComponent: React.FC<Props> = props => {
                 <TextFieldComponent name="participle" label="Participle" />
               </div>
             )}
-            {validated ? (
+            {validated && (secondAttempt || verbCorrect.all) ? (
               <>
-                <ShowResults succeeded={isCorrect} verb={verb} />
+                <ShowResults secondAttempt={true} verbCorrect={verbCorrect} verb={verb} />
```

Comment:
I guess this is true waiting for the setting to be added.

Right now add a TODO indicating that

```jsx
{/*TODO: pending on secondAttempt config setting*/}
```

Reply:
Yes, it is always true until we have implemented the settings storage.